cvpy/annotation/cvat/CVATTask.py

```diff
@@ -0,0 +1,48 @@
+import uuid
+from http import HTTPStatus
+
+import requests
+
+from cvpy.base.ImageTable import ImageTable
+from cvpy.annotation.base.Task import Task
+from cvpy.annotation.base.Project import Project
+
+
+class CVATTask(Task):
+    """ Defines a class to interact with with a CVAT Task.
+    
+    Parameters
+    ----------
+    start_image_id: 
```

Comment:
Can you explain to me the logic behind the start_image_id and end_image_id parameters?

Reply:
Sure! So CVAT calls them 'frame id's' for one - just a small thing to note as you're looking at CVAT compared to our code.  

When you upload images CVAT expects a list of files, well these ID's are effectively the list indexes. If you do a GET request to pull down an image you would need to pass it the ID (which is the index of the file in the list you originally uploaded).

Pankaj had a good suggestion to set these ID's in the constructor of a task, so these should be managed internal to the class from here on out.


Of note: you CAN override the ID's...like upload 5 images with id's 61 - 65. They seem to be more about ordering as far as CVAT is concerned than being the raw list indexes.